.github/workflows/test-and-build.yaml

```diff
@@ -26,34 +26,27 @@ jobs:
     needs: test
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
         uses: actions/checkout@v2
-      -
-        name: Get the tag name
+      - name: Get the tag name
         run: |
           echo "TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
-      -
-        name: Docker meta
+      - name: Docker meta
         id: meta
         uses: docker/metadata-action@v3
         with:
-          images: pihole/pihole
+          images: ${{ secrets.DOCKERHUB_NAMESPACE }}/pihole
```

Comment:
worth going one further and having `DOCKERHUB_REPO(NAME)`?, too?

Reply:
I don't know. I think that may end up having too many variables set. I know there was a user request for being able to build on their own forks so I guess if there's another request for changing the project name then we can look at it?